subworkflows/local/prepare_intervals.nf

```diff
@@ -35,46 +34,38 @@ workflow PREPARE_INTERVALS {
         ch_intervals_bed_gz_tbi = Channel.fromPath(file("${params.outdir}/no_intervals.bed.{gz,gz.tbi}"))
                                             .collect().map{ it -> [it, 0]}
 
-        ch_intervals_combined_bed_gz_tbi = Channel.fromPath(file("${params.outdir}/no_intervals.bed.{gz,gz.tbi}"))
-                                            .collect()
+        ch_intervals_combined = Channel.fromPath(file("${params.outdir}/no_intervals.bed"))
+                                            .map{ it -> [it, 0]}
```

Comment:
```suggestion
                                            .map{ it -> it[0]}
```
Do you mean this?

Reply:
no, the 0 is the `num_intervals` which is used throughout the pipeline to determine whether to do any parallelization or not.
Let me add a comment